packages/smarthr-ui/src/components/Dialog/DialogHeader.tsx

```diff
@@ -18,19 +18,19 @@ export type Props = {
   titleId: string
 }
 
-const dialogHeader = tv({
+const classNameGenerator = tv({
   base: [
     'smarthr-ui-Dialog-titleArea',
     'shr-border-b-shorthand shr-px-1.5 shr-py-1 shr-flex-[0_0_auto]',
   ],
 })
```

Comment:
ask: useMemo しなくてもいいのかなぁと思ったんだけど、一貫性を取ります?

```suggestion
const className = tv({
  base: [
    'smarthr-ui-Dialog-titleArea',
    'shr-border-b-shorthand shr-px-1.5 shr-py-1 shr-flex-[0_0_auto]',
  ],
})()
```

Reply:
この場合は一貫性取ったほうが良いかなぁ...

そもそもここ、tv使う必要性あります？
shorthandとかも関係ないし、variantsなどでの分岐もないし...
classNameにベタ書きでも良いのでは？
そしてそれならuseMemo消していいと思います